addon/styles/partials/_uni-button-dropdown.scss

```diff
@@ -0,0 +1,127 @@
+.uni-button-dropdown {
+  position: relative;
+
+  &--active {
+    .uni-button-dropdown__list {
+      display: block;
+    }
+
+    .uni-button-dropdown__button {
+      &:before {
+        transform: rotate(45deg);
+      }
+
+      &:after {
+        transform: rotate(-45deg);
+      }
+    }
+  }
+
+  &__button {
+    &:before, &:after {
+      content: " ";
+      position: absolute;
+      display: block;
+      width: 6px;
+      height: 2px;
+      top: calc(50% + 1px);
+      right: 8px;
+      background-color: #545d69;
+      transform: rotate(-45deg);
+      will-change: transform;
+    }
+
+    &:after {
+      right: 11px;
+      transform: rotate(45deg);
+    }
+
+    position: relative;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    font-family: "IntervalNextReg", sans-serif;
+    min-width: 130px;
+    height: 44px;
+    padding: 0 22px;
+    max-width: 180px;
+    background-color: transparent;
+    border-radius: 2px;
+    border: none;
+    outline: none;
+    will-change: background-color;
+    font-size: 1rem;
+    text-align: center;
+    color: #00adef;
+    min-width: initial;
+    height: 38px;
+    font-family: "IntervalNextReg", sans-serif;
+    font-size: 0.9375rem;
+    color: #545d69;
+  }
+
+  &__list {
+    position: absolute;
+    display: none;
+    width: 248px;
+    padding: 16px 0;
+    top: calc(100% + 11px);
+    right: 8px;
+    will-change: display;
+    backface-visibility: hidden;
+    background-color: #fff;
+    box-shadow: 0 16px 18px 0 rgba(31, 36, 50, 0.32);
+    border: solid 1px #ccd2d4;
+    border-radius: 2px;
+
+    &:before, &:after {
+      content: " ";
+      position: absolute;
+      display: block;
+      width: 0;
+      height: 0;
+      top: -7px;
+      right: 16px;
+      z-index: 2;
+      border-bottom: 8px solid #fff;
+      border-left: 8px solid transparent;
+      border-right: 8px solid transparent;
+    }
+
+    &:after {
+      top: -8px;
```

Comment:
Maybe it wasn't you, but this file indentation is all fucked up, CSS should be indented with 4 spaces